src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <title>Javascript Group Capstone Project By Misheck livingi and Ezekiel Utshudi  </title>
+</head>
+<body>
+  <header>
+    <div class="navbar">
+      <a class="link-logo" href="./index.html"> <img class="logo" src="../src/img/logo.png" alt="logo"></a>
+      <nav class="links">
+        <ul class="link-lists">
+          <li class="link-list"><a href="./index.html" class="link tv-show-link"></a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="main-content">
+    <div class="grid-cards-container"></div>
+  </main>
+  <aside class="movie-popup"></aside>
+  <footer>
+    Javascript Group Capstone By Misheck livingi and Ezekiel
+  </footer>
+</body>
```

Comment:
- [ ] Your Homepage footer is not displayed. Kindly fix that.

  ![ezekiel-footer](https://user-images.githubusercontent.com/36542986/163361950-764adb07-1143-42da-929b-c9bb84f05aa6.jpg)